draft-ietf-quic-qlog-main-schema.md

```diff
@@ -663,19 +663,20 @@ enable automated validation of aggregated qlog schemas.
 The only common field defined for the data field is the `trigger` field,
 which is discussed in {{trigger-field}}.
 
-~~~ cddl
+~~~
 ; The ProtocolEventData is any key-value map (e.g., JSON object)
-; only the optional trigger field is defined in this document
```

Comment:
I recall we had some previous discussion about this. I'd never acknowledged this trigger field (from a serializer or parser perspective). Are we aware of anyone using it?

Reply:
To clarify, up till now every instance of a `$ProtocolEventData`, such as parsing a H3 frame https://quicwg.org/qlog/draft-ietf-quic-qlog-h3-events.html#section-4.6, had an optional `trigger` field. This is not something I ever implemented for quiche. For serialization, that's no problem because the field can be omitted in logs. But for parsing, it is a problem, because I would not deal well with a qlog file that included such triggers.

That's my faul for omitting it from the quiche implementation in the first place. I support this change removing `trigger`, it means my implementation will be correct. However, anyone that is currently making use of `trigger` in event bodies may be surprised by this change. Hence the ask, is anyone using it?

If they are, it would be good to know if its inside a few events, or many. We can always add an explicit trigger to events where it makes sense. I don't think it makes sense for there to be a blanket trigger.

Reply:
Discussed during editors meeting: 
@LPardue will send an email to the mailing list detailing this change and asking for feedback before merging.

Reply:
Email was sent May 13th (https://mailarchive.ietf.org/arch/msg/quic/e4CnmK2_NUrNAHOlo9Q-fV1JW6g/) without comments from the WG. So it seems this should be a go.